rest/functionsapitest/graphql_test.go

```diff
@@ -469,6 +507,31 @@ func TestGraphQLQueryCustomUser(t *testing.T) {
 		assert.Equal(t, 200, response.Result().StatusCode)
 	})
 
+	//Check If User is not able to call Resolver within which Function  is not accessible to user
+	t.Run("User should receive error if the Function in Resolver does not belong to user channel", func(t *testing.T) {
```

Comment:
It can have a shorter name.
Something like: "AsUser - valid channel check"
The comment as it explains the entire usecase.